apps/test/unit/storage/FirebaseStorageTest.js

```diff
@@ -463,9 +464,14 @@ describe('FirebaseStorage', () => {
       );
 
       function deleteTable() {
-        FirebaseStorage.deleteTable('mytable', verifyNoTable, error => {
-          throw error;
-        });
+        FirebaseStorage.deleteTable(
```

Comment:
Does deleteTable accept 4 params or does it need to be updated?

Reply:
That's was leftover from when this was based off #30861, should be gone now